res/values/strings.xml

```diff
@@ -469,12 +469,19 @@
     <string name="confirmation_remove_folders_alert">Do you really want to remove the selected items and their contents?</string>
     <string name="failed_upload_quota_exceeded_text">Not enough space in the account</string>
     <string name="auth_forbidden">Forbidden authentication</string>
-
+    
     <string name="actionbar_privacy_policy">Privacy policy</string>
     <string name="prefs_privacy_policy">Privacy policy</string>
     <string name="privacy_policy_error">"An error has occurred: "</string>
     <string name="service_unavailable">Server is unavailable</string>
 
+    <string name="streaming_certificate_error">Streaming cannot be initialized because your server certificate is untrusted. File download will start automatically</string>
+    <string name="streaming_unrecognized_input">This video format is not supported</string>
+    <string name="streaming_file_not_found_error">This video file is no longer available in the server</string>
+    <string name="previewing_video_common_error">This video cannot be reproduced</string>
+
+    <string name="previewing_video_not_supported">Video previewing is not supported in your Android version. File download will start automatically and you will be able to open it with an external player</string>
```

Comment:
Yes, I prefer your error message by I don't agree with you about not showing the error message. It's the current behaviour in the released app but we should give feedback to the user as much as possible. If the user see that the download starts automatically, he will wonder why. 

ICE CREAM SANDWICH is only considered in PreviewVideoFragment because there is no way to go into full screen mode if the video player is not still prepared (full screen button is hidden till the video player is prepared)